js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -412,13 +412,20 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
       <div data-testid={testId('choose-partitions-dialog')}>
         <Warnings
           launchAsBackfill={launchAsBackfill}
-          isPureAssetBackfill={target.type === 'pureAssetBackfill'}
+          isPureAssetBackfill={target.backfills.type === 'pure'}
           upstreamAssetKeys={upstreamAssetKeys}
           selections={selections}
           setSelections={setSelections}
         />
         <ToggleableSection
-          title={<Subheading>Partition selection</Subheading>}
+          title={
+            <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
+              <Subheading>Partition selection</Subheading>
+              <span>
+                {keysFiltered.length} {keysFiltered.length === 1 ? 'partition' : 'partitions'}
+              </span>
+            </Box>
+          }
```

Comment:
Noticed this was missing @salazarm, I think maybe we forgot to put the count in?

Reply:
Oh I had taken the count after talking adhoc with @braunjj but I'm fine with adding it back